### ISSUE TITLE
The balances are not loading.

### ISSUE THREAD

**Issue by rafaelmadaloz** [amandris/arbitrage-trading-bot#8]
I am using the Bitstamp and Kraken APIs, with the correct keys, but the balance is not loading. Just get reset.

Follows the images of the program running on my local server.

![print1](https://user-images.githubusercontent.com/33348229/67678535-977bd900-f965-11e9-8b1d-970d9b5b13ed.png)

![print2](https://user-images.githubusercontent.com/33348229/67678571-b8442e80-f965-11e9-86c4-d2236522900f.png)

![print3](https://user-images.githubusercontent.com/33348229/67678572-b8442e80-f965-11e9-8b35-b581bcc53ddb.png)

**Comment by amandris** [comment]
Does clicking on the **refresh balances button** have any effect? Let me know if any value shows up or if there is any **js** error.

I have just tried it out with those same exchanges and it worked for me.

**Comment by rafamadaloz** [comment]
Clicking refresh just keeps loading forever, but never gives any results, nothing is displayed on the console.

![print4](https://user-images.githubusercontent.com/33348229/67800308-d002dc00-fa65-11e9-8f6f-25160ae9f4e3.png)

This print was taken after 30 seconds I clicked refresh

**Comment by amandris** [comment]
Please, check if you have the balance/funds permissions in your api settings page. These are the links:

https://www.bitstamp.net/account/security/api/

https://www.kraken.com/u/settings/api

**Comment by rafamadaloz** [comment]
I think everything is ok. This is my configuration

![Screenshot_2019-11-04-13-43-28](https://user-images.githubusercontent.com/33348229/68214897-0c31c180-ffbd-11e9-99ba-782f8cce9064.png)
![Screenshot_2019-11-04-13-43-25](https://user-images.githubusercontent.com/33348229/68214898-0cca5800-ffbd-11e9-979b-abf88c41bc95.png)

**Comment by z13gcarb** [comment]
Yes, those permissions seem to be ok. Do you get any error on the console output while running `bin/console bot:trade`?

**Comment by starloungeswitzerland** [comment]
I have the same error on click on **refresh balances button**
On my console:
```
Failed to load resource: the server responded with a status of 500 (Internal Server Error)
http://localhost:8000/balances-from-exchanges
```

```
Failed to load resource: the server responded with a status of 500 (Internal Server Error)
http://localhost:8000/difference
```

**Comment by z13gcarb** [comment]
@starlounge That error is from the js console. Is there any error on the Symfony log?

**Comment by starloungeswitzerland** [comment]
@z13gcarb in the Synfony log we have an error 500 for calls on:
- balances-from-exchanges
- difference

**Comment by Garcicasti** [comment]
@rafaelmadaloz  Try to run "composer install" again, when all the configuration files are already modified accordingly. I think this also happenned to me while installing it and it was because the installation didnt finish well at first

**Comment by starloungeswitzerland** [comment]
I have try to reinstall, make configuration and run again composer install, but after OK message for cache cleared I have this warning:

`Class SbS\AdminLTEBundle\Composer\ScriptHandler is not autoloadable, can not call symfony-scripts script`

**Comment by Sascha321** [comment]
> I have try to reinstall, make configuration and run again composer install, but after OK message for cache cleared I have this warning:
> 
> `Class SbS\AdminLTEBundle\Composer\ScriptHandler is not autoloadable, can not call symfony-scripts script`

Hello starloungeswitzerland
Hast Du das Tool zum laufen bekommen?

Gruss Sascha
